exampleCourse/questions/demo/cGrader/info.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
Maybe shift this question to `demo/autograder/c/square` to mirror `demo/autograder/python/square`?